### PR TITLE
Review SolrJ code for typos

### PR DESCRIPTION
**Pull request by epugh**
# Description

Go through and fix typos and grammer errors.

# Solution

Clicking on each java class in IntelliJ and looking for typos and grammer errors.  Using suggestions from IntelliJ.

I have fixed a couple of typos in method names where they were private to a class.

Since this is refactoring, not planning on a JIRA ticket..

# Tests

Please describe the tests you've developed or run to confirm this patch implements the feature or solves the problem.

# Checklist

Please review the following and check all that apply:

- [ ] I have reviewed the guidelines for [How to Contribute](https://github.com/apache/solr/blob/main/CONTRIBUTING.md) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [ ] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended, not available for branches on forks living under an organisation)
- [ ] I have developed this patch against the `main` branch.
- [ ] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

**Comment by epugh** [comment]
Anyone want to review 132 files ;-)

File: solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java [hunk 1]
```diff
@@ -121,7 +121,7 @@ protected ResponseParser responseParser(SolrRequest<?> solrRequest) {
     return solrRequest.getResponseParser() == null ? this.parser : solrRequest.getResponseParser();
   }
 
-  protected ModifiableSolrParams initalizeSolrParams(
+  protected ModifiableSolrParams initializeSolrParams(
```
Comment (janhoy) [inline solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java:124]: This one is prorected, meaning that potentially 3rd party sub classes could call it? However, how common would that be?

**Review by janhoy — COMMENTED** [review]
Thanks for caring for the small things. Browsed through, all text changes in comments look good. Just had one question regarding a protected method name change, which should perhaps go through deprecation...

File: solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java (hunk 1 above)
Comment (epugh) [reply solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java:124]: Dunno...   Would a 3rd party sub class immediately break and then you fix it?  I could easily do the deprecation however if you think that is just the easier process.....   Especailly for backporting...

**Comment by epugh** [comment]
Stuck in an airport on 5 hours of sleep and my brain was too dead to do anything more than mindless mousing and clicking!

File: solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java (hunk 1 above)
Comment (janhoy) [reply solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java:131]: I pushed the original method signature as deprecated. That deprecation can be removed in a followup commit to main after backporting. Or we could just do the plain rename and add in the deprecation when backporting to 9x?

Comment (epugh) [reply solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java:131]: i prefer to add the deprecation, and then do the main, branch_9x, and then come back and in a seperate commit remove the deprecated code.   Just to make it simpler to backport.   However, that might be me..  I've struggled on some of the backporting!

**Review by mkhludnev — APPROVED** [review]

File: solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java (hunk 1 above)
Comment (epugh) [reply solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java:131]: okay, in looking I see how you did it...!  All makes sense..      We need a plan for removing deprecated code in `main` anyway, there is so much of it!

**Pull request merged**